features/nanostack/mbed-mesh-api/source/WisunInterface.cpp

```diff
@@ -113,20 +116,42 @@ nsapi_error_t WisunInterface::configure()
                                             MBED_CONF_MBED_MESH_API_WISUN_BC_FIXED_CHANNEL,
                                             MBED_CONF_MBED_MESH_API_WISUN_BC_DWELL_INTERVAL,
                                             MBED_CONF_MBED_MESH_API_WISUN_BC_INTERVAL);
-    if (status < 0) {
+    if (status != MESH_ERROR_NONE) {
         tr_error("Failed to set broadcast channel function configuration");
         return NSAPI_ERROR_PARAMETER;
     }
 #endif
 
 #ifdef MBED_CONF_MBED_MESH_API_WISUN_NETWORK_SIZE
     status = set_network_size(MBED_CONF_MBED_MESH_API_WISUN_NETWORK_SIZE);
-    if (status < 0) {
+    if (status != MESH_ERROR_NONE) {
         tr_error("Failed to set network size");
         return NSAPI_ERROR_PARAMETER;
     }
 #endif
 
+#ifdef MBED_CONF_MBED_MESH_API_RADIUS_SHARED_SECRET
+    // Set on here, for compatibility with applications not using WisunBorderRouter class */
```

Comment:
Is there any such applications that can be used without the Border router interface? the Border router is not started without it

Reply:
There are applications, e.g. nanostack border router that use directly the nanostack c-api and does not use the border router class to start the BBR.

Reply:
Removed the changes from wisuninterface. Applications using RADIUS should use the wisunborderrouter class.